pipelines/pipelines/buildutils/build-branch.yaml

```diff
@@ -89,6 +89,7 @@ spec:
     runAfter:
     - clone-automation
     - make
+    - make-openapi2bean
     params:
     - name: pipelineRunName
```

Comment:
It's not letting me highlight the correct lines but lines 99-101 you should probably rename things to be `buildutils` not `galasabld` if it's no longer specific to `galasabld`.
You could make a directory in dockerfiles called `buildutils` for this and move it there.
Also rename the image to be called `buildutils-binary-downloadables` so it's more accurate - this will also require an update in a values.yaml file off the top of my head so do a search for "galasabld-binary-downloadables" to find where to update that

Reply:
ok, do you think it would be better to open up a new pull request for dockerfile specific stuff? and was going to open up a seperate pull request for changing the image name since stuff is likely to break if I do everything at once.

Reply:
for context, I can't remember what I did before for dockerfile changes :D